src/rpc/blockchain.cpp

```diff
@@ -379,18 +379,24 @@ void entryToJSON(UniValue &info, const CTxMemPoolEntry &e)
 {
     AssertLockHeld(mempool.cs);
 
+    UniValue fees(UniValue::VOBJ);
+    fees.push_back(Pair("current", ValueFromAmount(e.GetFee())));
```

Comment:
Not sure about `current`. Other options are `base`, `raw`, `original`, `self`...